src/server/game/Entities/Object/Object.h

```diff
@@ -21,6 +21,7 @@
 #include "Common.h"
 #include "GridReference.h"
 #include "GridRefManager.h"
+#include "Map.h"
```

Comment:
needs to be removed, `enum LineOfSightChecks` moved to SharedDefines.h (this is a change from include cleanup that was initially done in master then ported to 3.3.5, so it will not be picked anywhere else)

Reply:
Wouldn't it make sense to drop this when we port the SharedDefines.h commit to master? That's why I didn't bother to find a better temporary place for the enum.

Reply:
But you will not be porting that SharedDefines.h commit, thats why